frontend/src/App.js

```diff
@@ -0,0 +1,156 @@
+import React,{useState,useEffect} from 'react';
+import axios from 'axios'
+import './App.css';
+import Board from './Board'
+
+class App extends React.Component{
+constructor(props){
+    super(props)
+    this.handle_render = this.handle_render.bind(this)
+    this.state = {turn: "",board: null,count: 0}
+    }
+
+     handle_render(){
+        this.fetch_board_data()
+     }
+     create_new_board(){
+        fetch('http://localhost:5000/restart')
+        this.fetch_board_data()
+     }
+     fetch_board_data(){
+          try{
+             const fetch_data = async () => {
+                  await axios.get('http://localhost:5000/').then(
+                  response => {
+                     const data = response.data
+                     this.setState({turn:data.turn, board:data.board})
+                     console.log(this.state)
+                  })
+              }
+
+             fetch_data()
+          }catch(error){
+
+         }
+         }
```

Comment:
you can separate this component into two files:
one for all the fetching functions and one for just rendering.
you import the rendering component inside the logic component.
here's a nice example:"
https://dev.to/tomekbuszewski/high-level-view-and-logic-separation-in-react-39n0